docs/examples/tensorflow/tensorflow-various-readers.ipynb

```diff
@@ -245,6 +245,21 @@
     "import tensorflow as tf\n",
     "import nvidia.dali.plugin.tf as dali_tf\n",
     "\n",
+    "try:\n"
+    "    from tensorflow.compat.v1 import GPUOptions\n"
+    "    from tensorflow.compat.v1 import ConfigProto\n"
+    "    from tensorflow.compat.v1 import Session\n"
+    "except:\n"
+    "    # Older TF versions don't have compat.v1 layer\n"
+    "    from tensorflow import GPUOptions\n"
+    "    from tensorflow import ConfigProto\n"
+    "    from tensorflow import Session\n"
+    "\n"
+    "try:\n"
+    "    tf.compat.v1.disable_eager_execution()\n"
+    "except:\n"
+    "    pass\n"
+    "\n"
```

Comment:
I think this needs `,` at the end of each line for some reason. Otherwise there seems to be some JSON parsin failure in CI

Reply:
Good catch! Thanks